lisa/transformers/scripts_execution_transformer.py

```diff
@@ -0,0 +1,87 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional, Type
+
+from dataclasses_json import dataclass_json
+
+from lisa import schema
+from lisa.operating_system import Posix
+from lisa.transformers.deployment_transformer import (
+    DeploymentTransformer,
+    DeploymentTransformerSchema,
+)
+
+
+@dataclass_json()
+@dataclass
+class ScriptEntry:
+    script: str = ""
+    interpreter: str = ""
+    args: Optional[List[str]] = None
+
+
+@dataclass_json()
+@dataclass
+class ScriptsExecutionTransformerSchema(DeploymentTransformerSchema):
+    dependent_packages: List[str] = field(default_factory=list)
+    scripts: List[ScriptEntry] = field(default_factory=list)
+
+
+class ScriptExecutionResult:
```

Comment:
It's similar with the ExecutionResult. Does it need?

Reply:
updated